### PR TITLE
Update php-cs-fixer to 3.0.0

### PR DESCRIPTION
**Pull request by karyna-t**
### Description (*)
Update friendsofphp/php-cs-fixer to version 3.0.0

File with rules was renamed and adjusted according to this changelog:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.0.0/UPGRADE-v3.md

### Related Pull Requests
https://github.com/magento/partners-magento2ee/pull/572

### Fixed Issues (if relevant)
1. Fixes magento/magento2#33507: Update friendsofphp/php-cs-fixer composer dependency to version 3.0.0

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by karyna-tsymbal-atwix** [comment]
@magento run all tests

File: composer.lock [hunk 1]
```diff
@@ -5164,36 +5164,37 @@
         },
         {
             "name": "symfony/console",
-            "version": "v4.4.26",
+            "version": "v4.4.27",
```
Comment (ihor-sviziev) [inline composer.lock]: This version is buggy, please update to 4.4.29, or rollback
More info here https://github.com/magento/magento2/issues/33595

Comment (karyna-t) [reply composer.lock]: @ihor-sviziev  thank you for the info! I've updated PR, please re-check

**Comment by karyna-tsymbal-atwix** [comment]
@magento run all tests

**Comment by karyna-tsymbal-atwix** [comment]
@magento run Functional Tests B2B, Functional Tests EE

**Comment by karyna-tsymbal-atwix** [comment]
@magento run Functional Tests B2B

**Review by ihor-sviziev — APPROVED** [review]

**Review by andrewbess — APPROVED** [review]
Hello @karyna-tsymbal-atwix 
Thank you for your contribution.
Great job.

**Pull request merged**